pep-0556.rst

```diff
@@ -57,7 +57,7 @@ or physical) nodes.
 Interrupting application code at arbitrary points to execute finalization
 code that may rely on a consistent internal state and/or on acquiring
 synchronization primitives give rise to reentrancy issues that even the
```

Comment:
(Not related to your change, but noticed this typo.)
```suggestion
synchronization primitives gives rise to reentrancy issues that even the
```

Reply:
Thank you, will fix.